ci-operator/step-registry/quay-tests/aws-ipi-disconnected-private-deprovision/quay-tests-aws-ipi-disconnected-private-deprovision-chain.yaml

```diff
@@ -7,6 +7,5 @@ chain:
   - ref: quay-tests-recycle-omr
   - ref: aws-deprovision-stacks
   - ref: aws-deprovision-s3buckets
-  - ref: aws-deprovision-users-and-policies
```

Comment:
@LiZhang19817 @dongboyan77 this step has been moved into `cucushift-installer-rehearse-aws-ipi-deprovision`, please take a look and add lgtm label, thanks